src/Model/HasContentItemInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php declare(strict_types=1);
+/**
+ * @copyright Zicht Online <https://zicht.nl>
+ */
+
+namespace Zicht\Bundle\PageBundle\Model;
+
+/**
+ * Interface to indicate that an object (= parent) has a (child) ContentIntemInterface
+ */
+interface HasContentItemInterface
```

Comment:
Ik zou dit direct in de CC verwerken als deze PR gemerged, anders gaat niemand dit weten/gebruiken. Verder gebruiken we nog zelden globale contentitems in team 1, is dat voor team 2 anders? Anders is deze interface niet zo nuttig denk ik